src/main/java/seedu/address/ui/HelpContentManager.java

```diff
@@ -212,21 +222,14 @@ public String getContent(String key) {
      * @return An observable list with the section titles for the help window.
      */
     public ObservableList<String> getTableOfContents() {
-        return FXCollections.observableArrayList(
-                "Introduction",
-                "Quick Start",
-                "Command Format Guidelines",
-                "Adding a Contact",
-                "Listing All Contacts",
-                "Editing a Contact",
-                "Finding Contacts",
-                "Deleting a Contact",
-                "Clearing All Entries",
-                "Marking a Person as Paid/Unpaid",
-                "Saving and Editing Data",
-                "Exiting the Program",
-                "Command Summary"
-        );
+        ObservableList<String> tocList = FXCollections.observableArrayList(
+                "Introduction", "Quick Start", "Command Format Guidelines", "Adding a Contact",
+                "Listing All Contacts", "Editing a Contact", "Finding Contacts", "Deleting a Contact",
+                "Clearing All Entries", "Marking a Person as Paid/Unpaid", "Saving and Editing Data",
+                "Exiting the Program", "Command Summary");
```

Comment:
why the change in code design? thought the earlier version might make it clearer to see